docs/usage/config-presets.md

```diff
@@ -140,7 +140,7 @@ You do not need to add it as a devDependency or add any other files to the prese
 ## GitLab-hosted Presets
 
 It is also possible to host your preset config using just a regular GitLab repository and without needing to publish it to npmjs.
-In such cases Renovate will simply look for a renovate.json file in the default branch, (for now only the master branch is supported).
+In such cases Renovate will simply look for a `renovate.json` file in the default branch, (for now only the `master` branch is supported). ???TODO: Does GitLab now support `main`???
```

Comment:
I think that GitLab now supports `main` as well. But do we support the `main` branch on GitLab?

Reply:
Yes, supported

Reply:
```suggestion
In such cases Renovate will simply look for a `renovate.json` file in the default branch.
```